core/src/main/java/io/undertow/server/protocol/proxy/ProxyProtocolReadListener.java

```diff
@@ -103,9 +103,6 @@ public void handleEvent(StreamSourceChannel streamSourceChannel) {
         } catch (IOException e) {
             UndertowLogger.REQUEST_IO_LOGGER.ioException(e);
             IoUtils.safeClose(streamConnection);
-        } catch (Exception e) {
```

Comment:
I don't think we should be excluding this catch @baranowb

Reply:
hmm. brain fart. for some reason I assumed RE is not tied to E....